<?xml version="1.0" encoding="UTF-8"?><testrun name="EstadoServiceTests" project="Hackathon" tests="3" started="3" failures="0" errors="1" ignored="0">
  <testsuite name="com.iadtec.hackathon.service.EstadoServiceTests" time="16.077">
    <testcase name="contextLoads" classname="com.iadtec.hackathon.service.EstadoServiceTests" time="0.272"/>
    <testcase name="createPais" classname="com.iadtec.hackathon.service.EstadoServiceTests" time="15.795">
      <error>org.springframework.jdbc.UncategorizedSQLException: PreparedStatementCallback; uncategorized SQLException for SQL [SELECT ESTADO.ID, ESTADO.NOME, ESTADO.UF, PAIS.NOME AS PAIS FROM ESTADO INNER JOIN PAIS ON PAIS.ID = ESTADO.PAIS WHERE NOME=? AND ROWNUM = 1 ]; SQL state [90059]; error code [90059]; Ambiguous column name "NOME"; SQL statement:
SELECT ESTADO.ID, ESTADO.NOME, ESTADO.UF, PAIS.NOME AS PAIS FROM ESTADO INNER JOIN PAIS ON PAIS.ID = ESTADO.PAIS WHERE NOME=? AND ROWNUM = 1  [90059-197]; nested exception is org.h2.jdbc.JdbcSQLException: Ambiguous column name "NOME"; SQL statement:
SELECT ESTADO.ID, ESTADO.NOME, ESTADO.UF, PAIS.NOME AS PAIS FROM ESTADO INNER JOIN PAIS ON PAIS.ID = ESTADO.PAIS WHERE NOME=? AND ROWNUM = 1  [90059-197]
	at org.springframework.jdbc.support.AbstractFallbackSQLExceptionTranslator.translate(AbstractFallbackSQLExceptionTranslator.java:89)
	at org.springframework.jdbc.support.AbstractFallbackSQLExceptionTranslator.translate(AbstractFallbackSQLExceptionTranslator.java:81)
	at org.springframework.jdbc.support.AbstractFallbackSQLExceptionTranslator.translate(AbstractFallbackSQLExceptionTranslator.java:81)
	at org.springframework.jdbc.core.JdbcTemplate.translateException(JdbcTemplate.java:1444)
	at org.springframework.jdbc.core.JdbcTemplate.execute(JdbcTemplate.java:632)
	at org.springframework.jdbc.core.JdbcTemplate.query(JdbcTemplate.java:669)
	at org.springframework.jdbc.core.JdbcTemplate.query(JdbcTemplate.java:700)
	at org.springframework.jdbc.core.JdbcTemplate.query(JdbcTemplate.java:712)
	at org.springframework.jdbc.core.JdbcTemplate.queryForObject(JdbcTemplate.java:783)
	at com.iadtec.hackathon.estado.EstadoService.findByNome(EstadoService.java:63)
	at com.iadtec.hackathon.service.EstadoServiceTests.createPais(EstadoServiceTests.java:28)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.springframework.test.context.junit4.statements.RunBeforeTestExecutionCallbacks.evaluate(RunBeforeTestExecutionCallbacks.java:74)
	at org.springframework.test.context.junit4.statements.RunAfterTestExecutionCallbacks.evaluate(RunAfterTestExecutionCallbacks.java:84)
	at org.springframework.test.context.junit4.statements.RunBeforeTestMethodCallbacks.evaluate(RunBeforeTestMethodCallbacks.java:75)
	at org.springframework.test.context.junit4.statements.RunAfterTestMethodCallbacks.evaluate(RunAfterTestMethodCallbacks.java:86)
	at org.springframework.test.context.junit4.statements.SpringRepeat.evaluate(SpringRepeat.java:84)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.runChild(SpringJUnit4ClassRunner.java:251)
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.runChild(SpringJUnit4ClassRunner.java:97)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
	at org.springframework.test.context.junit4.statements.RunBeforeTestClassCallbacks.evaluate(RunBeforeTestClassCallbacks.java:61)
	at org.springframework.test.context.junit4.statements.RunAfterTestClassCallbacks.evaluate(RunAfterTestClassCallbacks.java:70)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.run(SpringJUnit4ClassRunner.java:190)
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:89)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:41)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:541)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:763)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:463)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:209)
Caused by: org.h2.jdbc.JdbcSQLException: Ambiguous column name "NOME"; SQL statement:
SELECT ESTADO.ID, ESTADO.NOME, ESTADO.UF, PAIS.NOME AS PAIS FROM ESTADO INNER JOIN PAIS ON PAIS.ID = ESTADO.PAIS WHERE NOME=? AND ROWNUM = 1  [90059-197]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:357)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.expression.ExpressionColumn.mapColumn(ExpressionColumn.java:127)
	at org.h2.expression.ExpressionColumn.mapColumns(ExpressionColumn.java:97)
	at org.h2.expression.Comparison.mapColumns(Comparison.java:510)
	at org.h2.expression.ConditionAndOr.mapColumns(ConditionAndOr.java:260)
	at org.h2.expression.ConditionAndOr.mapColumns(ConditionAndOr.java:260)
	at org.h2.command.dml.Select.mapColumns(Select.java:1240)
	at org.h2.command.dml.Select.init(Select.java:826)
	at org.h2.command.Parser.parseSelect(Parser.java:1925)
	at org.h2.command.Parser.parsePrepared(Parser.java:463)
	at org.h2.command.Parser.parse(Parser.java:335)
	at org.h2.command.Parser.parse(Parser.java:307)
	at org.h2.command.Parser.prepareCommand(Parser.java:278)
	at org.h2.engine.Session.prepareLocal(Session.java:611)
	at org.h2.engine.Session.prepareCommand(Session.java:549)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1247)
	at org.h2.jdbc.JdbcPreparedStatement.&lt;init&gt;(JdbcPreparedStatement.java:76)
	at org.h2.jdbc.JdbcConnection.prepareStatement(JdbcConnection.java:304)
	at com.zaxxer.hikari.pool.ProxyConnection.prepareStatement(ProxyConnection.java:311)
	at com.zaxxer.hikari.pool.HikariProxyConnection.prepareStatement(HikariProxyConnection.java)
	at org.springframework.jdbc.core.JdbcTemplate$SimplePreparedStatementCreator.createPreparedStatement(JdbcTemplate.java:1557)
	at org.springframework.jdbc.core.JdbcTemplate.execute(JdbcTemplate.java:615)
	... 37 more

</error>
    </testcase>
    <testcase name="deletePais" classname="com.iadtec.hackathon.service.EstadoServiceTests" time="0.005"/>
  </testsuite>
</testrun>
